package.json

```diff
@@ -7,9 +7,10 @@
     "lerna-changelog": "cli.js"
   },
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/__mocks__/GithubAPI.js,src/__mocks__/Changelog.js",
```

Comment:
The problem is I keep those mocked folders is that they will be present in `/lib` as well as `/src` and when `jest` runs it outputs some logs. Unfortunately I couldn't find a way of configuring this with `jest`, if you know of a better solution I'm all ears.

```bash
jest-haste-map: duplicate manual mock found:
  Module name: GithubAPI
  Duplicate Mock path: /Users/emmenko/dev/src/lerna-changelog/src/__mocks__/GithubAPI.js
This warning is caused by two manual mock files with the same file name.
Jest will use the mock file found in:
/Users/emmenko/dev/src/lerna-changelog/src/__mocks__/GithubAPI.js
 Please delete one of the following two files:
 /Users/emmenko/dev/src/lerna-changelog/lib/__mocks__/GithubAPI.js
/Users/emmenko/dev/src/lerna-changelog/src/__mocks__/GithubAPI.js


jest-haste-map: duplicate manual mock found:
  Module name: Changelog
  Duplicate Mock path: /Users/emmenko/dev/src/lerna-changelog/src/__mocks__/Changelog.js
This warning is caused by two manual mock files with the same file name.
Jest will use the mock file found in:
/Users/emmenko/dev/src/lerna-changelog/src/__mocks__/Changelog.js
 Please delete one of the following two files:
 /Users/emmenko/dev/src/lerna-changelog/lib/__mocks__/Changelog.js
/Users/emmenko/dev/src/lerna-changelog/src/__mocks__/Changelog.js
```